src/components/DeleteCars/DeleteList.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import EachCarDelete from './EachCarDelete';
+
+const DeleteList = () => {
+  const cars = useSelector((state) => state.cars.cars);
+  //   const mockData = [
+  //     {
+  //         "id": 3,
+  //         "name": "Car1",
+  //         "description": "Description for Car1",
+  //         "removed": false
+  //     },
+  //     {
+  //         "id": 4,
+  //         "name": "Car1111",
+  //         "description": "Description for Car1",
+  //         "removed": false
+  //     }
+  // ];
+  // const cars = mockData;
+
```

Comment:
-[ ] Kindly, delete the old commented code present in this file in order to preserve the repo's cleanliness. 👍